clarity/src/vm/docs/mod.rs

```diff
@@ -2261,6 +2262,7 @@ This function returns the STX balance, in microstacks (1 STX = 1,000,000 microst
 const STX_GET_ACCOUNT: SimpleFunctionAPI = SimpleFunctionAPI {
     name: None,
     snippet: "stx-account ${1:owner}",
+    output_type: "{locked: uint, unlock-height: uint, unlocked: uint}",
```

Comment:
I would instead use this for output_type: "(tuple (locked uint) (unlock-height uint) (unlocked uint))"